cadence/server/public/js/aria.js

```diff
@@ -0,0 +1,197 @@
+var streamSrcURL = "" // this is used by the stream source loader
+
+$(window).on("load", function(e) {
+	getRadioData()
+	connectToSSE()
+	postSearch()
+	setSearchRequestFunctions()
+});
+
+function getRadioData() {
+	$.ajax({
+		type: 'GET',
+		url: "/api/version",
+		dataType: "json",
+		success: function(data) {
+			document.getElementById("release").innerHTML = data.Version;
+		},
+		error: function() {
+			document.getElementById("release").innerHTML = "(N/A)";
+		}
+	});
+	$.ajax({
+		type: 'GET',
+		url: "/api/nowplaying/metadata",
+		dataType: "json",
+		success: function(data) {
+			$('#song').text(data.Title)
+			$('#artist').text(data.Artist)
+		},
+		error: function() {
+			$('#song').text("-")
+			$('#artist').text("-")
+		}
+	});
+	$.ajax({
+		type: 'GET',
+		url: "/api/nowplaying/albumart",
+		dataType: "json",
+		success: function(data) {
+			$('#artwork').attr("src", "data:image/jpeg;base64," + data.Picture);
+		},
+		error: function() {
+			$('#artwork').attr("src", "");
```

Comment:
Couldn't we hardcode an SVG error (single character in Unicode) sort of thing here?

I suppose that's not necessarily better...